### DISCUSSION TITLE
Profile for STALKER

### DISCUSSION THREAD

**Discussion by sbstn87** [AntiMicroX/antimicrox#887] [Show and tell]
Hi guys,
I have created a profile for STALKER:
https://www.gog.com/game/stalker_shadow_of_chernobyl

How can I upload it to be available on the profiles page?

**Comment by pktiuk** [comment]
Profiles can be uploaded in this repository https://github.com/AntiMicroX/antimicrox-profiles

![image](https://github.com/AntiMicroX/antimicrox/assets/45544416/36da7f97-30bc-4544-a650-43268b1c2016)
